index.html

```diff
@@ -20,38 +20,35 @@
 <body>
   
   <div class="grid-container">
-  
     <header>
-      <div class="logo">
-        <a href="#">
-          <img src="images/icon.png" alt="JT icon">
-        </a>
-      </div>
-      <div class="name">
-        <a href="#">Jud&aacute; Teixeira</a>
-      </div>
-      <div class="menu-mobile">
-        <nav>
-          <a href="#">
-            <img src="images/menu.png" alt="Menu">
-          </a>
-        </nav>
-      </div>
-      <div class="menu-desktop">
-        <nav>
-          <a href="#Portfolio">
-            Portfolio
-          </a>
-          <a href="#About">
-            About
-          </a>
-          <a href="#Contact">
-            Contact
-          </a>
-        </nav>
+      <div class="name-logo">
+        <a id="name-logo" href="#">Jud&aacute; Teixeira</a>
       </div>
+      <nav class="menu-mobile-container hide">
```

Comment:
- [ ] Great job! You did almost everything in this project. I just want you to do more to get your mobile menu very identical to that of Figma's. 
![image](https://user-images.githubusercontent.com/45668340/147113467-84882207-b794-4a16-b767-e6db96ab9cf0.png)
## Yours
![image](https://user-images.githubusercontent.com/45668340/147113621-5703874f-53cb-4992-bede-ebb23ec22f3d.png)
- Could you please **blur** the background so it looks like the Figma design? You can do that by adding something like `backdrop-filter: blur(8px);` to the overlay element.
- You can prevent the body from scrolling vertically when the menu pops up.
![new](https://user-images.githubusercontent.com/45668340/147114518-d493478c-6e24-4cd7-b86a-ac2b474e928e.gif)
You may try adding this class to the body when the menu pops up:
```
  height: 100%;
  overflow: hidden;
  ```

Reply:
Done, thank you!